m1well.Expenses/src/expenses.js

```diff
@@ -362,43 +362,46 @@ const fixedTracking = async (): Promise<boolean> => {
 }
 
 /**
- * provide config from _configuration and cast content to real objects
+ * provide config from new plugin settings section or the old _configuration file and cast content to real objects
  *
  * @private
  */
-const provideConfig = (): Promise<Config> => {
-  return getOrMakeConfigurationSection(
-    'expenses',
-    EXAMPLE_CONFIG
-  )
-    .then(result => {
-      if (result == null || Object.keys(result).length === 0) {
-        logError('expected config could not be found in the _configuration file')
-        return {
-          folderPath: '',
-          delimiter: '',
-          dateFormat: '',
-          amountFormat: '',
-          columnOrder: [],
-          categories: [],
-          shortcutExpenses: [],
-          fixedExpenses: []
-        }
-      } else {
-        logMessage(`loaded config\n${JSON.stringify(result)}\n`)
-        const config: Config = {
-          folderPath: castStringFromMixed(result, CONFIG_KEYS.folderPath),
-          delimiter: castStringFromMixed(result, CONFIG_KEYS.delimiter),
-          dateFormat: castStringFromMixed(result, CONFIG_KEYS.dateFormat),
-          amountFormat: castStringFromMixed(result, CONFIG_KEYS.amountFormat),
-          columnOrder: castStringArrayFromMixed(result, CONFIG_KEYS.columnOrder),
-          categories: castStringArrayFromMixed(result, CONFIG_KEYS.categories),
-          shortcutExpenses: castShortcutExpensesArrayFromMixed(result, CONFIG_KEYS.shortcutExpenses),
-          fixedExpenses: castFixedExpensesArrayFromMixed(result, CONFIG_KEYS.fixedExpenses),
-        }
-        return config
+const provideConfig = async (): Promise<Config> => {
+  const fromSettings: Config = DataStore.settings
+
+  if (fromSettings && fromSettings.useNewSettings) {
```

Comment:
It's up to you, but none of the other plugins have the equivalent of 'useNewSettings'.